src/Adapter/SsiCache.php

```diff
@@ -140,23 +120,14 @@ public function cacheAction(Request $request)
             $this->argumentResolver->getArguments($subRequest, $controller) :
             $this->resolver->getArguments($subRequest, $controller);
 
-        // call controller
-        return call_user_func_array($controller, $arguments);
+        return \call_user_func_array($controller, $arguments);
```

Comment:
Why the prefix? You didn't prefix all internal componentent (e.g. scalar types)

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048#issuecomment-328810072 and rest of the list https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048#issuecomment-332784692

Reply:
also https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/

Reply:
I know that this are some minor performance improvents, but didn't notice that this is automatically done with the CS fixer.

Reply:
I am not sure if ours does it automatically but I added it, @jordisala1991 mentioned [here](https://github.com/sonata-project/SonataCacheBundle/pull/205#discussion_r168935431), should I remove it?

Reply:
You dont have to manually add them, if the php-cs-fixer adds them its fine, if not I would say not add anything

Reply:
In doctrine, they would `use function call_user_func_array` at the top of the file, that's another way to do it.